pkgs/applications/audio/spotify-cli/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, python3Packages, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "spotify-cli";
+  version = "0.3.12";
+
+  src = fetchFromGitHub {
+    owner = "ledesmablt";
+    repo = pname;
+    rev = "8e0ad476079a722a39dd2072195c6b4562d18a4d";
+    sha256 = "sha256-KSlv+O1XQPFrRyxx8/+lCzWryp+9aXpB5CvhEM9w1wA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    inquirerpy
+    tabulate
+  ];
+
+  preBuild = ''
+    export RELEASE_VERSION=${version}
+  '';
+
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
Are you willing to maintain this package? If you are, [you should list yourself as a maintainer](https://github.com/NixOS/nixpkgs/blob/dc1a82101ced2daf22d079d9bcc596531ab9ebac/maintainers/README.md). If you aren't, [then this probably shouldn't be added to nixpkgs](https://github.com/NixOS/nixpkgs/blob/dc1a82101ced2daf22d079d9bcc596531ab9ebac/pkgs/README.md?plain=1#L34-L36).